### ISSUE TITLE
Add configuration directive for RequestedAuthnContext

### ISSUE THREAD

**Issue by pandafy** [IdentityPython/pysaml2#806]
Adding a configuration directive for `RequestedAuthnContext` will simplify usage and solve https://stackoverflow.com/questions/61834932/pysaml2-authncontextclassref-passwordprotectedtransport

**Comment by pandafy** [comment]
Should the necessary changes for this go under `pysaml2.src.saml2.config.Config` or `pysaml2.src.saml2.config.SPConfig`?

**Comment by peppelinux** [comment]
Afaik SPConfig because SP Is the only entity that makes authnrequests

I've not checked the code, does It look good to you?

**Comment by pandafy** [comment]
> I've not checked the code, does It look good to you?

Yes! Currently, i am just making changes here and there trying to make it work.

**Comment by peppelinux** [comment]
See tests/

We have a specific test file with all the examples to get It working
